src/pybito.cpp

```diff
@@ -21,8 +21,9 @@ namespace py = pybind11;
 // Thanks to @eacousineau!
 template <typename PyClass, typename C, typename D>
 void def_read_write_mutable(PyClass &cls, const char *name, D C::*pm) {
-  cls.def_property(name, [pm](C & self) -> auto & { return self.*pm; },
-                   [pm](C &self, const D &value) { self.*pm = value; });
+  cls.def_property(
```

Comment:
This change was caused by clang-format.  I assume we want it updated?

Reply:
Yeah, sure. I'll see if my clang-format reverts it somehow.